backend/app/main.py

```diff
@@ -1,19 +1,24 @@
-from typing import Union
-
+import psycopg2
 from fastapi import FastAPI
+import Product
 
 app = FastAPI()
 
+#establishing the connection
+conn = psycopg2.connect(
+   database="fridgeai_db", user='fridgeai_user', password='fridgeai_password', host='localhost', port= '5432'
+)
 
-@app.get("/")
-def read_root():
-    return {"Hello": "World"}
-
-@app.get("/hey")
-def read_root():
-    return {"Hey": "Omg Hi!"}
-
+#Setting auto commit false
+conn.autocommit = True
 
-@app.get("/items/{item_id}")
-def read_item(item_id: int, q: Union[str, None] = None):
-    return {"item_id": item_id, "q": q}
+#Creating a cursor object using the cursor() method
+cursor = conn.cursor()
+@app.get("/products")
+def Get_List_Of_Products():
```

Comment:
```suggestion
def Get_List_Of_Products():
```
def get_products_list():